sdk/ml/azure-ai-ml/azure/ai/ml/_utils/_logger_utils.py

```diff
@@ -19,16 +20,18 @@ def initialize_logger_info(module_logger: logging.Logger, terminator="\n") -> No
     module_logger.addHandler(handler)
 
 
+tracer = trace.get_tracer(AML_INTERNAL_LOGGER_NAMESPACE)
+
+
 class OpsLogger:
     def __init__(self, name: str):
-        self.package_logger: logging.Logger = logging.getLogger(AML_INTERNAL_LOGGER_NAMESPACE + name)
-        self.package_logger.propagate = False
-        self.package_tracer = None
+        self.package_logger: logging.Logger = logging.getLogger(AML_INTERNAL_LOGGER_NAMESPACE).getChild(name)
+        self.package_logger.propagate = True
```

Comment:
What difference does the boolean change here make? For parent + child operations?

Reply:
In the current design, we have a parent logger on which distro is configured and child loggers for all operations classes. Setting propagate to True here makes sure that the logs on these child loggers are passed to the parent for handling and exporting. And on parent, we have set propagate to False to make sure it doesn't go any further up.